Userland/Libraries/LibWeb/Layout/LayoutState.cpp

```diff
@@ -584,6 +588,9 @@ void LayoutState::UsedValues::set_content_width(CSSPixels width)
 
 void LayoutState::UsedValues::set_content_height(CSSPixels height)
 {
+    dbgln("Debug: height value before clamping: {}", height);
+    if (height.might_be_saturated())
+        height = max_height;
     VERIFY(!height.might_be_saturated());
```

Comment:
might_be_saturated() check is intentional. hitting this verify, means something went wrong in formatting context run and we want to figure what exactly instead of covering it up.